webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -174,6 +174,14 @@ class HttpGroovyTest implements HttpConfiguration {
         }
     }
 
+    @Test
+    void "query params in url"() {
+        http.get("params?a=1&b=text") {
```

Comment:
just to be mean: what happens if you repeat the "a" key, e.g. `params?a=1&a=x`

Reply:
The test service used by these tests works on matching url and the one that's been setup for this is `/params?a=1&b=text`.  I could add one with repeated params but that's not a realistic test of a "real" service.

That said, I'm pretty sure the http code in webtau (and in HttpUrlConnection) will handle that correctly if listed in the url string.  I don't think webtau's `HttpQueryParams` class can handle that though.  If we feel it should, I suggest we tackle it in a separate PR.